### PR TITLE
[diem-transactional-tests] migrate tests -- 2/N

### PR DESCRIPTION
**Pull request by vgao1996**
This migrates a bunch of old functional tests and makes a few minor enhancements to the transactional test framework:
1. (Diem) Admin scripts now takes two signers properly.
2. (Diem) It's now possible to specify the remaining transaction parameters as extra args.
3. (Diem/Move) Named addresses can now be used as transaction arguments. 
4. (Diem/Move) `call_function` and `execute_script` now return `Result<Option<String>>` instead of `Result<()>`, making it possible to emit additional texts to the exp file.
5. (Diem) The `run` command now gets an optional flag `--show-events`, which will print out the events emitted if present.
6. (Diem) Implemented a pretty printer for emitted events.

Each of these enhancements can be found in its own commit. Same for tests in each folder (category).

I've also left a bunch of `// TODO`s to highlight the tests with oddities. They don't have to be dealt with right now, but we should take a closer look at them when we have time, understand their intentions and see if we need to improve/rewrite them.

### CODE REVIEW

File: language/testing-infra/transactional-test-runner/src/tasks.rs [hunk 1]
```diff
@@ -409,3 +417,11 @@ impl FromStr for PrintBytecodeInputChoice {
         }
     }
 }
+
+fn parse_argument(s: &str) -> Result<Argument> {
+    Ok(if let Some(stripped) = s.strip_prefix('@') {
```
Comment (tnowacki) [inline language/testing-infra/transactional-test-runner/src/tasks.rs:422]: nit, please switch to match :)

Comment (vgao1996) [reply language/testing-infra/transactional-test-runner/src/tasks.rs:422]: Good catch. Will do in a future commit.

**Review by tnowacki — APPROVED** [review]

**Pull request closed without merging**